common/network-shuffle/src/test/java/org/apache/spark/network/shuffle/ExternalShuffleBlockResolverSuite.java

```diff
@@ -118,27 +112,6 @@ public void testSortShuffleBlocks() throws IOException {
   }
 
   @Test
-  public void testHashShuffleBlocks() throws IOException {
-    ExternalShuffleBlockResolver resolver = new ExternalShuffleBlockResolver(conf, null);
```

Comment:
If I'm not mistaken, I think that this code path is exercised by the external shuffle service, so a Spark 2.x shuffle service would be unable to serve shuffle files generated by Spark 1.x when using hash shuffle.